src/main/resources/bridge-sdk.properties

```diff
@@ -48,6 +48,7 @@ survey.close.api = /researchers/v1/surveys/%s/%s/close
 ### Upload Schemas
 uploadschema.api = /researcher/v1/uploadSchema
 uploadschema.by.id.api = /researcher/v1/uploadSchema/byId/%s
+uploadschema.by.id.and.rev.api = /researcher/v1/uploadSchema/byIdAndRev/%s/%d
```

Comment:
I know we've discussed this, but I think the "by*" path elements are not required or standard for resource-oriented paths, this is clear:

/researcher/v1/uploadSchema/<id>
/researcher/v1/uploadSchema/<id>/<revision>